assets/css/startertheme.css

```diff
@@ -0,0 +1,114 @@
+body {
+  color: #555;
+  font-family: Georgia, "Times New Roman", Times, serif;
+  height:100%;
+}
+
+h1 {
+  margin-top: 0;
+  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: normal;
+  text-align: left;
+}
+
+/*
+ * Override Bootstrap's default container.
+ */
+
+@media (min-width: 1200px) {
+  .container {
+    width: 970px;
+
+  }
+}
+.carousel-indicators li {
+  border-radius: 50%;
+  margin: 1px 3px;
+  max-width: 15px;
+  background-color: transparent;
+}
+
+.carousel-inner .item img{
+  width: 100%;
+  margin: auto;
+  height: 400px;
+  text-align: center;
+}
+
+.row .col-xs {
+  display: grid;
+  justify-content: space-between;
+}
+
+.hovereffect {
+  width:100%;
+  height:100%;
+  float:left;
+  overflow:hidden;
+  position:relative;
+  text-align:left;
+  cursor:default;
+  }
+
+  .hovereffect .overlay {
+  width:100%;
+  height:100%;
+  position:absolute;
+  overflow:hidden;
+  top:0;
+  left:0;
+  opacity:0;
+  background-color: white;
+  -webkit-transition:all .4s ease-in-out;
+  transition:all .4s ease-in-out;
+  }
+
+  .hovereffect .large-img {
+  width: 200px;
+  height: 200px;
+  display:block;
+  position:relative;
+  -webkit-transition:all .4s linear;
+  transition:all .4s linear;
+
+  }
+
+ .overlay .small-img{
+  display: inline-block;
+  width: 30%;
+  /* position:relative; */
+  -webkit-transition:all .4s linear;
+  transition:all .4s linear;
+  }
+
+  .hovereffect .title {
+    font-size: 16px;
+    text-align: center;
+  }
+
+  .overlay .content {
+    font-size: 14px;
+    width: 200px;
+    height: 200px;
+    border-bottom: 2px solid #555;
+    overflow: scroll;
+  }
+
+  .hovereffect button {
+    text-decoration:none;
+    display:inline-block;
+    background-color: #FF6969;
+    color: #2E2A2B;
+    height: 15%;
+    text-align: center;
+    border-radius: 3px;
+    cursor: pointer;
+    margin-left: 10%;
+    padding-left: 5%;
+  }
+
+  .hovereffect:hover .overlay {
+  opacity:1;
+  }
+
```

Comment:
For this file, the font-families and colours should have been set up in bootstrap for the most part. When you're doing width and height, try your best to use `rem` instead of `px`. Before you change width and height you should try finding bootstrap classes that will do it for you, ex. `w-100` makes it `width: 100%`.